appendices/special-characters/search-place.md

```diff
@@ -1,34 +1,34 @@
 # Search Place
 
-## **Syntax**
+## Syntax
 
-```text
+```G1ANT
 ✱part of text✱
 ```
 
-## **Description**
+## Description
 
-Search place character ****`✱`  is used within the text.find command to specify location for text search. If only a part of the text to find is known, or you wish to find ANYTHING in the text, this special character shall highlight the ANYTHING.
+Search place character `✱`  is used within the text.find command to specify location for text search. If only a part of the text to find is known, or you wish to find ANYTHING in the text, this special character shall highlight the ANYTHING.
 
 You can insert this character from `Insert/Search Place` menu or with **Ctrl+8** keyboard shortcut.
 
-### **Example 1**
+### Example 1
 
 You want to search for a text stored in a variable, and get everything between texts `Name:` and `Surname:`. 
 
-```text
+```G1ANT
 ♥source = ‴Name: John Surname: Smith‴
 text.find ♥source search Name:✱Surname: result ♥name
 dialog ♥name
 ```
 
 ![img](../../-assets/search1_v1.png)
 
-**Example 2**
+### Example 2
 
-When there are more elements matching your query, the result is an array \(a list\). More on arrays [here](array-separator.md).
+When there are more elements matching your query, the result is an array (a list). More on arrays [here](array-separator.md).
```

Comment:
is in a list*
more on array separators*
let's not confuse arrays with lists in G1ANT. I know we have array separator now but let's not call lists arrays :)

Reply:
```suggestion
When there are more elements matching your query, the result is a list. More on lists [here](array-separator.md).
```